test/service/test_manifest.py

```diff
@@ -241,6 +241,30 @@ def sort_key(row: JSON) -> bytes:
         expected.sort(key=sort_key)
         self.assertEqual(expected, manifest)
 
+    def _assert_pfb(self,
```

Comment:
If we extract a method, we should simplify the contract between caller and callee. This method has side-effects on an argument, which is generally frowned upon, unless performance considerations make that necessary, in which case the docstring should make that clear. In this case, we should avoid the side-effects, either by not sorting, or at least not sorting in place.